dotCMS/src/main/java/com/dotmarketing/cms/myaccount/struts/MyAccountForm.java

```diff
@@ -8,6 +8,7 @@
 import com.dotcms.repackage.org.apache.struts.action.ActionMessage;
 import com.dotmarketing.util.UtilMethods;
 
+@Deprecated
 public class MyAccountForm extends ActionForm
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)